examples/hybrid/sphere/hadley_circulation.jl

```diff
@@ -0,0 +1,331 @@
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+    return (hv_center_space, hv_face_space)
+end
+
+# RHS tendency specification
+function rhs!(dydt, y, parameters, t, alpha, beta)
+
+    (; coords, face_coords, τ, u₀, ω₀, K, ρ₀, ystar) = parameters
+
+    ϕ = coords.lat
+    zc = coords.z
+    zf = face_coords.z
+    ϕf = face_coords.lat
+
+    uu = @. u₀ * cosd(ϕ)
+    uv = @. -(R * ω₀ * pi * ρ₀ / K / z_top / ρ_ref(zc)) *
+       sind(K * ϕ) *
+       cosd(ϕ) *
+       cos(pi * zc / z_top) *
+       cos(pi * t / τ)
+    uw = @. (ω₀ * ρ₀ / K / ρ_ref(zf)) *
+       (-2 * sind(K * ϕf) * sind(ϕf) + K * cosd(ϕf) * cosd(K * ϕf)) *
+       sin(pi * zf / z_top) *
+       cos(pi * t / τ)
+
+    uₕ = Geometry.Covariant12Vector.(Geometry.UVVector.(uu, uv))
+    w = Geometry.Covariant3Vector.(Geometry.WVector.(uw))
+
+    # aliases
+    ρ = y.ρ
+    ρq1 = y.ρq1
+    dρq1 = dydt.ρq1
+
+    # No change in density: divergence-free flow
+    @. dydt.ρ = beta * dydt.ρ + 0 .* y.ρ
+
+    If2c = Operators.InterpolateF2C()
+    Ic2f = Operators.InterpolateC2F(
+        bottom = Operators.Extrapolate(),
+        top = Operators.Extrapolate(),
+    )
+    vdivf2c = Operators.DivergenceF2C(
+        top = Operators.SetValue(Geometry.Contravariant3Vector(0.0)),
+        bottom = Operators.SetValue(Geometry.Contravariant3Vector(0.0)),
+    )
+    third_order_upwind_c2f = Operators.Upwind3rdOrderBiasedProductC2F(
+        bottom = Operators.ThirdOrderOneSided(),
+        top = Operators.ThirdOrderOneSided(),
+    )
+    hdiv = Operators.Divergence()
+    hwdiv = Operators.WeakDivergence()
+    hgrad = Operators.Gradient()
+
+    ### HYPERVISCOSITY
+    @. ystar.ρq1 = hwdiv(hgrad(ρq1 / ρ))
+    Spaces.weighted_dss!(ystar.ρq1)
+    @. ystar.ρq1 = -κ₄ * hwdiv(ρ * hgrad(ystar.ρq1))
+
+    cw = If2c.(w)
+    cuvw = Geometry.Covariant123Vector.(uₕ) .+ Geometry.Covariant123Vector.(cw)
+
+    @. dρq1 = beta * dρq1 - alpha * hdiv(cuvw * ρq1) + alpha * ystar.ρq1
+    @. dρq1 -= alpha * vdivf2c(Ic2f(ρ) * third_order_upwind_c2f.(w, ρq1 ./ ρ))
+    @. dρq1 -= alpha * vdivf2c(Ic2f(uₕ * ρq1))
+
+    Spaces.weighted_dss!(dydt.ρ)
+    Spaces.weighted_dss!(dydt.ρq1)
+
+    return dydt
+end
+
+const R = 6.37122e6        # radius
+const grav = 9.8           # gravitational constant
+const R_d = 287.058        # R dry (gas constant / mol mass dry air)
+const z_top = 1.2e4        # height position of the model top
+const p_top = 25494.4      # pressure at the model top
+const T₀ = 300             # isothermal atmospheric temperature
+const H = R_d * T₀ / grav  # scale height
+const p₀ = 1.0e5           # reference pressure at the surface
+const ρ₀ = p₀ / R_d / T₀   # density at the surface
+const τ = 86400.0          # period of motion (1 day)
+const K = 5                # number of overturning cells
+const u₀ = 40.0            # reference zonal velocity
+const ω₀ = 0.15            # reference vertical velocity
```

Comment:
```suggestion
const w₀ = 0.15            # reference vertical velocity
```
(Just a minor suggestion, this is more consistent with the reference)